ios/Exponent/Versioned/Core/UniversalModules/EXNotifications/EXScopedNotificationsHandlerModule.m

```diff
@@ -0,0 +1,30 @@
+// Copyright 2018-present 650 Industries. All rights reserved.
+
+#import "EXScopedNotificationsHandlerModule.h"
+#import "EXScopedNotificationsUtils.h"
+
+@interface EXScopedNotificationsHandlerModule ()
+
+@property (nonatomic, strong) NSString *experienceId;
+
+@end
+
+@implementation EXScopedNotificationsHandlerModule
+
+- (instancetype)initWithExperienceId:(NSString *)experienceId
+{
+  if (self = [super init]) {
+    _experienceId = experienceId;
+  }
+  
+  return self;
+}
+
+- (void)userNotificationCenter:(UNUserNotificationCenter *)center willPresentNotification:(UNNotification *)notification withCompletionHandler:(void (^)(UNNotificationPresentationOptions))completionHandler
```

Comment:
Should we also override other methods and apply the same if logic in case someone some day adds some logic to the original class?